README.md

```diff
@@ -163,7 +163,7 @@ Notes:
 
 ## Contributing
 
-See [CONTRIBUTE.md](CONTRIBUTE.md)
+See [CONTRIBUTING.md](https://github.com/amzn/ion-js/blob/master/CONTRIBUTING.md)
```

Comment:
Thanks for noticing and correcting this.  Please change to:

`See [CONTRIBUTING.md](CONTRIBUTING.md)`

Reply:
Sure! Just changed it.